docs(main): add new image (#42)
fanux/sealos-site: src/components/Homepage/Examples/_examples_block.mdx

$ sealos run docker.io/labring/kubernetes-dashboard:v1.0.8

# How to access

> # You can run this command by local kubectl

Reviewer: ```suggestion
# You can run this command with local kubectl
```

```

```mdx-code-block
</TabItem>
<TabItem value="rancher" label="Rancher">
```

```shell

> ## how to use

Reviewer: ```suggestion
```

# Install ingress-nginx first, rancher need it
$ sealos run \
  --masters 192.168.72.50,192.168.72.51,192.168.72.52 \
  --nodes 192.168.72.53 -p 123456 \
  labring/kubernetes:v1.23.8 \
  labring/calico:v3.22.1 \
  labring/helm:v3.8.2 \
  labring/ingress-nginx:4.1.0 \
  labring/cert-manager:v1.8.0

> # Wait above ready，then run rancher

Reviewer: ```suggestion
# After the above is ready, run rancher
```

$ sealos run registry.cn-shenzhen.aliyuncs.com/cnmirror/rancher:v2.6.6

```

```mdx-code-block
</TabItem>
<TabItem value="kubesphere" label="Kubesphere">
```

```shell

> ## how to use

Reviewer: ```suggestion
```

> # prequires：openebs or other storage is needed by kubespere.

Reviewer: ```suggestion
# prerequisites: OpenEBS or other storage is needed by KubeSphere.
```
